packages/lib/src/link/Link.tsx

```diff
@@ -79,7 +79,6 @@ const StyledLink = styled.div<{
   border: none;
   border-radius: 4px;
   width: fit-content;
-  padding: 0 2px;
```

Comment:
Even though padding is not adequate because of the extra spacing between words when using links, I think `line-height` and `outline-offset` should be used to maintain the margin on focus as it is shown in the docs.

Reply:
I'm gonna review this with the designers!

Reply:
Updated!